fs_s3fs/_s3fs.py

```diff
@@ -221,6 +221,13 @@ class S3FS(FS):
         PyFilesystem specification exactly. Set to ``False`` to disable
         validation of destination paths which may speed up uploads /
         downloads.
+    :param str cache_control: Sets the 'Cache-Control' header for uploads.
+    :param str acl: Sets the Access Control List header for uploads.
+    :param dict upload_args: A dictionary for additional upload arguments.
```

Comment:
@geoffjukes Do you want to elaborate on this, or does this cover it?

Reply:
That about covers it @willmcgugan